snap/snapcraft.yaml

```
@@ -128,10 +128,46 @@ parts:
       - percona-server-mongodb
       - percona-backup-mongodb
       - util-linux
+    override-prime: |
+      craftctl default
+      gzip -df $CRAFT_PRIME/licenses/LICENSE-Community.txt.gz
+      mv $CRAFT_PRIME/licenses/LICENSE-Community.txt \
+        $CRAFT_PRIME/licenses/LICENSE-percona-server
+    organize:
+      usr/share/doc/percona-server-mongodb/LICENSE-Community.txt.gz:
+        licenses/LICENSE-Community.txt.gz
```

maybe also works, a bit simpler 
```suggestion
    override-prime: |
      craftctl default
      zcat $CRAFT_PRIME/usr/share/doc/percona-server-mongodb/LICENSE-Community.txt.gz \
        > $CRAFT_PRIME/licenses/LICENSE-percona-server
```